tck/features/expressions/temporal/Temporal5.feature

```diff
@@ -28,12 +28,10 @@
 
 #encoding: utf-8
 
-Feature: TemporalAccessorAcceptance
+Feature: Temporal5 - Access Parts of Temporal Values
```

Comment:
I think we have called these _Parts_ "_Components_": https://github.com/opencypher/openCypher/blob/master/cip/1.accepted/CIP2015-08-06-date-time.adoc#accessing-components-of-temporal-values

So I'd call this feature "Accessing Components of Temporal Values".

Reply:
```suggestion
Feature: Temporal5 - Access Components of Temporal Values
```

Yes, great suggestion. Will adopt. Thx.